vmck/settings.py

```diff
@@ -25,3 +27,8 @@
 
 _ssh_username = SSH_USERNAME
 SSH_USERNAME = os.environ.get('SSH_USERNAME', _ssh_username)
+
+sentry_sdk.init(
+    dsn="https://cfc9c776b9834e078b54b96f6826a0f6@sentry.io/1467831",
```

Comment:
Yeah, no, let's not publish our DSN. Please pull the value from an environment variable named `SENTRY_DSN`.